### PR TITLE
feat: add points command

### PR DESCRIPTION
**Pull request by ramirezfernando**
### Notes
- Add `/points` command with optional discord mention argument with added logic specified in [Bot Info](https://docs.google.com/document/d/1AUv2IV02YQ0cCoBzJoU2Z1vE1HNxDZibkyhZ9dZiMGI/edit)

### Screenshots
Screen Shot 2023-12-12 at 4 42 15 PM
Screen Shot 2023-12-12 at 4 43 36 PM

### Todo
- Add `/points lb` command

### CODE REVIEW

File: src/commands/points.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                .then(res => {
+                    return res.json()
+                })
+                .then(data => {
+                    if (data.success) {
+                        interaction.editReply(data.data.first_name + " " + data.data.last_name + ", you have " + data.data.points + " points");
+                    }
+                    else {
+                        interaction.editReply("Error: " + data.error.message);
+                    }
```
Comment (bnleft) [inline src/commands/points.ts:73]: **refactor:** let's use template literals and avoid using if-else

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: src/commands/points.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                .setDescription("Mention a discord user to view their member points!")
+                .setRequired(false)),
+    execute: async ({ interaction })=> {
+        try {
+            await interaction.deferReply({ ephemeral: false });
+    
+            const roleName = interaction.options.get("mention")?.role?.name
+            const mentionedEveryoneOrHere = (roleName == "@everyone") || (roleName == "@here")
+            if (mentionedEveryoneOrHere) {
+                await interaction.editReply("Don't do that!")
```
Comment (bnleft) [inline src/commands/points.ts:33]: <3

**Review by bnleft — CHANGES_REQUESTED** [review]

File: src/commands/points.ts (hunk 1 above)
Comment (ramirezfernando) [reply src/commands/points.ts]: Fixed in commit 2

File: src/commands/points.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                    "Content-Type": "application/json"
+                }
+            }
+            fetch(url, options)
+                .then(res => {
+                    return res.json()
+                })
+                .then(data => {
+                    if (data.success) {
+                        interaction.editReply(`${data.data.first_name} ${data.data.last_name}, you have ${data.data.points} points"`);
```
Comment (bnleft) [inline src/commands/points.ts:69]: **suggestion:** remove the double quotation mark

**Review by bnleft — CHANGES_REQUESTED** [review]

File: src/commands/points.ts (hunk 3 above)
Comment (ramirezfernando) [reply src/commands/points.ts]: Good catch, was removed in commit #3.

**Review by bnleft — APPROVED** [review]

**Pull request merged**